.github/workflows/cicd.yml

```diff
@@ -39,4 +54,4 @@ jobs:
           secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           region: us-east-1
           local-image: hdf5-to-cog
-          image: hdf5-to-cog
\ No newline at end of file
+          image: hdf5-to-cog
```

Comment:
@abarciauskas-bgse @slesaad 

Is this build/deploy step still valid?  

I see that we're running `docker build docker/hdf5-to-cog --tag hdf5-to-cog`, but `docker/hdf5-to-cog` doesn't seem to exist in this repo.

Reply:
yeah this is not valid now